keras_cv/metrics/coco/numerical_tests/mean_average_precision_test.py

```diff
@@ -39,7 +39,7 @@ class MeanAveragePrecisionTest(tf.test.TestCase):
     Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.610
     """
 
-    def test_mean_average_precision_correctness_default(self):
+    def DISABLE_test_mean_average_precision_correctness_default(self):
```

Comment:
If we're disabling all of these tests, we should also mark the API as private or at least add a docstring that it isn't stable

Reply:
Done